tools/c7n_org/c7n_org/cli.py

```diff
@@ -570,6 +589,16 @@ def accounts_iterator(config):
              'tags': a.get('tags', ()),
              'vars': _update(a.get('vars', {}), org_vars)}
         yield d
+    for a in config.get("tenancies", ()):
+        d = {"account_id": a["profile"],
+             "name": a.get("name", a["profile"]),
+             "regions": a.get("regions", ["global"]),
+             "provider": "oci",
+             "profile": a["profile"],
+             "tags": a.get("tags", ()),
+             "oci_compartments": a.get("vars").get("oci_compartments") if a.get("vars") else None,
```

Comment:
a.get("vars", {}) to skip the conditional at the end

Reply:
Done